<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataFilesPage</class>
 <widget class="QWidget" name="DataFilesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>557</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="dataTab">
      <attribute name="title">
       <string>Content Files</string>
      </attribute>
      <layout class="QGridLayout" name="dataTabLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="contentSelectorWidget" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="dataNoteLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Note: content files that are not part of current Content List are &lt;span style=&quot; font-style:italic;font-weight: bold&quot;&gt;highlighted&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dirTab">
      <attribute name="title">
       <string>Data Directories</string>
      </attribute>
      <layout class="QGridLayout" name="dirTabLayout">
       <item row="0" column="0">
        <widget class="QListWidget" name="directoryListWidget">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="directoryButtons">
         <item>
          <widget class="QPushButton" name="directoryAddSubdirsButton">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>33</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Scan directories for likely data directories and append them at the end of the list.</string>
           </property>
           <property name="text">
            <string>Append</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="directoryInsertButton">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>33</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Scan directories for likely data directories and insert them above the selected position</string>
           </property>
           <property name="text">
            <string>Insert Above</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="directoryUpButton">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>33</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Move selected directory one position up</string>
           </property>
           <property name="text">
            <string>Move Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="directoryDownButton">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>33</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Move selected directory one position down</string>
           </property>
           <property name="text">
            <string>Move Down</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="directoryRemoveButton">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>33</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Remove selected directory</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="directoryButtonsSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="directoryNoteLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Note: directories that are not part of current Content List are &lt;span style=&quot; font-style:italic;font-weight: bold&quot;&gt;highlighted&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="archiveTab">
      <attribute name="title">
       <string>Archive Files</string>
      </attribute>
      <layout class="QGridLayout" name="archiveTabLayout">
       <item row="0" column="0">
        <widget class="QListWidget" name="archiveListWidget">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::CopyAction</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="archiveButtons">
         <item>
          <widget class="QPushButton" name="archiveUpButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>33</verstretch>
            </sizepolicy>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>33</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Move selected archive one position up</string>
           </property>
           <property name="text">
            <string>Move Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="archiveDownButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>33</verstretch>
            </sizepolicy>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>33</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Move selected archive one position down</string>
           </property>
           <property name="text">
            <string>Move Down</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="archiveButtonsSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="archiveNoteLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Note: archives that are not part of current Content List are &lt;span style=&quot; font-style:italic;font-weight: bold&quot;&gt;highlighted&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="navigationMeshCacheTab">
      <attribute name="title">
       <string>Navigation Mesh Cache</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="navigationMeshCacheProgressLayout">
         <item>
          <widget class="QPushButton" name="updateNavMeshButton">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="toolTip">
            <string>Generate navigation mesh cache for all content. Will be used by the engine to make cell loading faster.</string>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="navMeshProgressBar">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelNavMeshButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Cancel navigation mesh generation. Already processed data will be saved.</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="navMeshRemoveUnusedTilesCheckBox">
         <property name="text">
          <string>Remove Unused Tiles</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="navigationMeshCacheParamsLayout">
         <item>
          <widget class="QLabel" name="navMeshMaxSizeLabel">
           <property name="text">
            <string>Max Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="navMeshMaxSizeSpinBox">
           <property name="suffix">
            <string> MiB</string>
           </property>
           <property name="maximum">
            <number>2147483647</number>
           </property>
           <property name="value">
            <number>2048</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="navMeshLogPlainTextEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="profileGroupBox">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="title">
      <string>Content List</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="ProfilesComboBox" name="profilesComboBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select a content list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="newProfileButton">
        <property name="toolTip">
         <string>New Content List</string>
        </property>
        <property name="text">
         <string>&amp;New Content List</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="cloneProfileButton">
        <property name="toolTip">
         <string>Clone Content List</string>
        </property>
        <property name="text">
         <string>Clone Content List</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deleteProfileButton">
        <property name="toolTip">
         <string>Delete Content List</string>
        </property>
        <property name="text">
         <string>Delete Content List</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="newProfileAction">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New Content List</string>
   </property>
   <property name="toolTip">
    <string>New Content List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="cloneProfileAction">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Clone Content List</string>
   </property>
   <property name="toolTip">
    <string>Clone Content List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="deleteProfileAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Delete Content List</string>
   </property>
   <property name="toolTip">
    <string>Delete Content List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="checkAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Check Selection</string>
   </property>
  </action>
  <action name="uncheckAction">
   <property name="text">
    <string>Uncheck Selection</string>
   </property>
  </action>
  <action name="refreshDataFilesAction">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Refresh Data Files</string>
   </property>
   <property name="toolTip">
    <string>Refresh Data Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProfilesComboBox</class>
   <extends>QComboBox</extends>
   <header>apps/launcher/utils/profilescombobox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
